server/channels/api4/ai.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package api4
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+func (api *API) InitAI() {
+	// GET /api/v4/ai/agents
```

Comment:
Do we want to have these under an `ai` route? Just worried that its a bit general for no specific reason. 
Maybe just `/api/v4/agents` and `/api/v4/llmservices`?

Also in general I like to avoid calling stuff AI we call them Agents not AI Agents.

Reply:
I think in this context `ai` makes sense because it encapsulates both Services and Agents. I know that AI can be a bit of a misnomer but for the average person looking to implement an AI feature, they're going to look for the AI controller imo.